package.json

```diff
@@ -13,7 +13,7 @@
     "format:verify": "format-files -i -style=file -verify",
     "postinstall": "yarn build",
     "spellcheck": "npx cspell",
-    "test": "lage test --verbose",
+    "test": "lage test --verbose --passWithNoTests",
```

Comment:
what does this do?

Reply:
When any of the "test" scripts run but they have no tests then the lage test command fails. The --passWithNoTests command lets the test suite pass when there are no tests.

Reply:
was this happening before or did something change?

Reply:
It looks like @react-native/monorepo just added a test script to their package.json, so we weren't running yarn test on that directory before.